pages/authPage/logout.html

```diff
@@ -12,15 +12,15 @@
 <body>
```

Comment:
UI Changes: For some reason, the navbar is broken do check it out!
![image](https://user-images.githubusercontent.com/67889207/157886299-c2f12ba5-e514-4902-a740-a49862bebaef.png)

Reply:
Navbar component is broken due to changes in styles CDN.
this is fixed in the next PR with react.